.github/workflows/rng.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const crypto = require('crypto');
+const seedrandom = require('./seedrandom');
+
+function genSeedText() {
+    const { RNG_SEED } = process.env;
+    if (RNG_SEED) {
+        // Explicit seed is specified so use it
+        return RNG_SEED;
+    }
+    const { GITHUB_PR_NUMBER } = process.env;
+    if (GITHUB_PR_NUMBER) {
+        // GitHub pull request so use the GITHUB_REF as the seed to ensure consistency throughout PR lifecycle
+        return 'pr_' + GITHUB_PR_NUMBER;
+    }
+    // Otherwise generate a random seed. Note that we do not actually care
+    return 'seed_' + Date.now() + '_' + crypto.randomBytes(16).toString('hex');
```

Comment:
What is the use of `Date` when using `crypto.randomBytes(..)`? The comment looks incomplete

Reply:
Including the current time does not make it any less random.